src/diag.rs

```diff
@@ -269,7 +271,7 @@ impl DiagnosticCode {
         use strum::IntoEnumIterator;
         crate::advisories::Code::iter()
             .map(Self::Advisory)
-            .chain(crate::bans::Code::iter().map(Self::Bans))
+            //.chain(crate::bans::Code::iter().map(Self::Bans))
```

Comment:
Was this meant to remain commented? I just noticed a failure from `--allow duplicate` no longer parsing.

Reply:
Oops, this was not, sorry.